pootle/apps/virtualfolder/urls.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django.conf.urls import include, url
+
+from pootle_store.urls import unit_xhr_urlpatterns
+
+from .views import get_vfolder_units, VFolderTPTranslateView
+
+
+vfolder_urlpatterns = [
+    # TP Translate
+    url(r'(?P<language_code>[^/]*)/'
```

Comment:
its more reusable to put the prefix in the include  - see below

that way the views can be used with other/out prefix etc